src/canvas-extractor/edfi_canvas_extractor/mapping/assignments.py

```diff
@@ -122,6 +122,8 @@ def map_to_udm_assignments(
         "LMSSectionSourceSystemIdentifier"
     ].astype("string")
 
+    assignments_df = assignments_df.drop_duplicates()
```

Comment:
Is this actually necessary?

Reply:
Yes... If you remove it, it duplicates records